.changeset/seven-seahorses-cry.md

```diff
@@ -0,0 +1,6 @@
+---
+"@preact/signals-core": minor
```

Comment:
This might need to be a major though 😅

Reply:
Minor for a type correction is completely a-ok, IMO.

Reply:
I'll expand on the changeset then to provide our reasoning to avoid excessive issues about it